chain-tx-enclave/mock-utils/src/lib.rs

```diff
@@ -0,0 +1,50 @@
+use parity_scale_codec::{Decode, Encode};
+
+use chain_core::state::tendermint::BlockHeight;
+use chain_core::tx::{data::TxId, PlainTxAux, TxObfuscated, TxWithOutputs};
+
+const ENCRYPTION_KEY: u8 = 0x0f;
+const SEAL_KEY: u8 = 0xf0;
+pub fn decryt(payload: &TxObfuscated) -> PlainTxAux {
+    let unpad = unpad_payload(&payload.txpayload);
+    let bs = unpad.iter().map(|b| b ^ ENCRYPTION_KEY).collect::<Vec<_>>();
+    PlainTxAux::decode(&mut bs.as_slice()).unwrap()
+}
+
+fn unpad_payload(payload: &[u8]) -> &[u8] {
+    &payload[0..payload.len() - 16]
+}
+
+pub fn seal(tx: &TxWithOutputs) -> Vec<u8> {
+    tx.encode()
+        .into_iter()
+        .map(|b| b ^ SEAL_KEY)
+        .collect::<Vec<_>>()
```

Comment:
There's now this pure Rust sealing: https://github.com/crypto-com/chain/blob/master/chain-tx-enclave/enclave-utils/src/lib.rs
instead of this custom, it could just produce mock payloads that correspond to the Intel SDK payloads, but it won't be deriving key using the "egetkey" instruction -- it'll just have one hardcoded mock key

Reply:
I feel that simple xor may be good enough for the mock purpose?

Reply:
these were just small suggestions -- the main requested change is just updating the changelog; it's ok to merge this without this improved mocking for now